<style type="text/css">
.tg  {border-collapse:collapse;border-spacing:0;}
.tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
  overflow:hidden;padding:10px 5px;word-break:normal;}
.tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
.tg .tg-f7xd{background-color:#CECECE;border-color:inherit;color:#333;font-weight:bold;text-align:left;vertical-align:top}
.tg .tg-kr4b{background-color:#FFF;border-color:inherit;color:#333;font-weight:bold;text-align:left;vertical-align:top}
.tg .tg-acii{background-color:#FFF;border-color:inherit;color:#333;text-align:left;vertical-align:top}
.tg .tg-prhl{background-color:#FFF;border-color:inherit;color:#1f2328;text-align:left;text-decoration:underline;vertical-align:top}
.tg .tg-07pr{background-color:#FFF;border-color:inherit;color:#1F2328;text-align:left;vertical-align:top}
.tg .tg-e4b2{background-color:#FFF;border-color:inherit;color:#1F1F1F;text-align:left;vertical-align:top}
.tg .tg-0pky{border-color:inherit;text-align:left;vertical-align:top}
</style>
<table class="tg" style="undefined;table-layout: fixed; width: 100%">
<colgroup>
<col style="width: 7%">
<col style="width: 20%">
<col style="width: 20%">
<col style="width: 40%">
<col style="width: 13%">
</colgroup>
<thead>
  <tr>
    <!-- <th class="tg-f7xd"><span style="font-weight:bold">Date</span></th> -->
    <th class="tg-f7xd"><span style="font-weight:bold">Title</span></th>
    <th class="tg-f7xd"><span style="font-weight:bold">Speaker</span></th>
    <th class="tg-f7xd"><span style="font-weight:bold">Content</span></th>
    <th class="tg-f7xd"><span style="font-weight:bold">Links</span></th>
  </tr>
</thead>
<tbody>
  <tr>
    <!-- <td class="tg-acii"><span style="color:#333;background-color:#FFF">Week 1 - 24.02</span></td> -->
    <td class="tg-acii"><span style="color:#333;background-color:#FFF;font-weight:bold">Models for multimodal data integration</span></td>
    <td class="tg-acii"><span style="color:#333;background-color:#FFF">Britta Velten (Heidelberg)</span></td>
    <td class="tg-acii">This lecture will provide an overview of the basic statistical concepts that are important for the joint analysis of multi-modal omics data with a focus on probabilistic models for data integration. We will discuss statistical properties of multi-omics data and the resulting challenges for the data analysis, followed by an overview on different strategies for both supervised and unsupervised integration. Taking MOFA as example for an unsupervised method we will discuss the properties of probabilistic factor models for joint dimension reduction of multiple omics data sets. We will also discuss avenues to account for multiple sample groups and omics data with temporal and spatial resolution in probabilistic models. </td>
    <td class="tg-acii"><a href='https://drive.google.com/drive/folders/10E9hFJ-4IU5dFqp9MIjXulWxKqt7btGK?usp=sharing'>Slides, notebooks and recording</a> </td>
  </tr>
  <tr>
    <!-- <td class="tg-acii"><span style="color:#333;background-color:#FFF">Week 2 - 3.03</span></td> -->
    <td class="tg-acii"><span style="color:#333;background-color:#FFF;font-weight:bold">VAE in single-cell genomics</span></td>
    <td class="tg-acii"><span style="color:#333;background-color:#FFF">Carl Herrmann (Heidelberg)</span></td>
    <td class="tg-acii">In this lecture, I will review recent applications of AE and VAE in the field of genomics, in particular single-cell genomics. We will see how these application can help perform clustering of cell populations and allow to denoise sparse data. Finally, I will present some recent VAE models which are interpretable, i.e. in which the neurons of the model can be interpreted as biological entities. For those not familiar with gemomics, I will start with a brief review of some concepts and data types.</td>
    <td class="tg-acii"><a href='https://drive.google.com/drive/folders/1h8P6r4tiTtA1lNr6OmjndmckvTSW1RNy?usp=sharing'>Slides, notebooks and recording</a> </td>
  </tr>
  <tr>
    <!-- <td class="tg-acii"><span style="color:#333;background-color:#FFF">Week 3 - 10.03</span></td> -->
    <td class="tg-acii"><span style="color:#333;background-color:#FFF;font-weight:bold">Deep learning for predicting non-coding DNA activity</span></td>
    <td class="tg-acii"><span style="color:#333;background-color:#FFF">Bartek Wilczynski (Warsaw)</span></td>
    <td class="tg-acii">We know that the transcriptional gene regulation in multicellular
      organisms depends on the action of hundreds of thousands non-coding
      regulatory sequences scattered in the genome. Since there are so many
      of them, and we ususally cannot assess their activity directly in the
      cells and tissues, annotating their activity experimentally in full is
      difficult.  If we simplify their function into activation of
      transcription in different cellular contexts, the task of annotation
      becomes similar to a classical ML problem of multi-class
      classification. Recently, many studies have been published attempting
      to solve this problem to different degrees using Convolutional Neural
      Networks. We will discuss a few recent such papers and discuss their
      successes as well as some pitfalls of training classifier models
      without clearly defined negative examples.
      </td>
    <td class="tg-acii"><a href='https://drive.google.com/drive/folders/1RMXgMYgDagVHC94VMizBXqdtZLdMzPya?usp=sharing'>Slides, notebooks and recording</a> </td>
  </tr>
  <tr>
    <td class="tg-acii" colspan="5"></td>
  </tr>
  <tr>
    <!-- <td class="tg-acii"><span style="color:#333;background-color:#FFF">Week 4 - 17.03</span></td> -->
    <td class="tg-acii"><span style="color:#333;background-color:#FFF;font-weight:bold">AlphaFold, EMSFold to predict structure of proteins</span></td>
    <td class="tg-acii"><span style="color:#333;background-color:#FFF">Joanna Sulkowska (Warsaw)</span></td>
    <td class="tg-acii"></td>
    <td class="tg-acii"><a href='https://drive.google.com/drive/folders/1WeZr9Z7ICl695aMU7VkqUPWFzErb2kJv?usp=sharing'>Slides, notebooks and recording</a></td>
  </tr>
  <tr>
    <!-- <td class="tg-acii"><span style="color:#333;background-color:#FFF">Week 5 - 24.03</span></td> -->
    <td class="tg-acii"><span style="color:#333;background-color:#FFF;font-weight:bold">Protein design in the deep learning era, from inverse folding to diffusion models</span></td>
    <td class="tg-acii"><span style="color:#333;background-color:#FFF">Elodie Laine (Paris)</span></td>
    <td class="tg-acii">The revolution in protein structure prediction has boosted the development of deep learning-based methods for designing de novo protein with desired properties. This session will introduce the modern computational pipeline for protein design, from generating novel protein folds to designing amino acid sequences compatible with them. We will cover a wide range of deep learning architectures and frameworks, including graph neural networks and diffusion models. We will discuss a few recent groundbreaking applications that were unimaginable just a few years ago. </td>
    <td class="tg-acii"><a href='https://drive.google.com/drive/folders/1kqAvnvuT4dMEzMVXAt2s_Pt14DGbX0iW?usp=sharing'>Slides and notebook</a></td>
  </tr>
  <tr>
    <!-- <td class="tg-acii"><span style="color:#333;background-color:#FFF">Week 6 - 31.03</span></td> -->
    <td class="tg-acii"><span style="color:#333;background-color:#FFF;font-weight:bold">Deep Architectures for sampling molecules</span></td>
    <td class="tg-acii"><span style="color:#333;background-color:#FFF">Grégoire Sergeant-Perthuis (Paris)</span></td>
    <td class="tg-acii">In this lecture, we will explain how to reduce and analyze molecular dynamics trajectories to cluster molecular conformations into discrete states and characterize the resulting conformational transitions..</td>
    <td class="tg-acii"><a href='https://drive.google.com/drive/folders/1eqav8H_-ATiJoMZbMvDVaugWY8LGZKSg?usp=sharing'>Slides and notebooks</a></td>
  </tr>
  <tr>
    <!-- <td class="tg-acii"><span style="color:#333;background-color:#FFF">Week 7 - 7.04</span></td> -->
    <td class="tg-acii"><span style="color:#333;background-color:#FFF;font-weight:bold">Deep learning models for protein-ligand binding site prediction</span></td>
    <td class="tg-acii"><span style="color:#333;background-color:#FFF">David Hoksza (Prague)</span></td>
    <td class="tg-acii">We will introduce methods for predicting protein-small molecule binding sites, with a focus on structure-based approaches. We will briefly cover traditional non-ML and ML methods, followed by deep learning techniques such as convolutional neural networks, graph neural networks, and the latest addition to the field—protein language models.
    </td>
    <td class="tg-acii"><a href='https://drive.google.com/drive/folders/1EMmL6iJQAgofSMUcGjTcR7mS2gSTqkz6?usp=sharing'>Slides, notebooks and recording</a></td>
  </tr>
  <tr>
    <td class="tg-acii" colspan="5"></td>
  </tr>
  <tr>
    <!-- <td class="tg-acii"><span style="color:#333;background-color:#FFF">Week 8 - 14.04</span></td> -->
    <td class="tg-acii"><span style="color:#333;background-color:#FFF;font-weight:bold">Deep learning for image segmentation</span></td>
    <td class="tg-acii"><span style="color:#333;background-color:#FFF">Karl Rohr (Heidelberg)</span></td>
    <td class="tg-acii">The lecture introduces deep learning methods for image segmentation. The focus is on Convolutional Neural Networks (CNNs) and encoder-decoder network architectures for cell segmentation. We will discuss the well-known networks U-Net and Cellpose, and their application for computer-based analysis of cell microscopy image data.</td>
    <td class="tg-acii"><a href="https://drive.google.com/drive/folders/1ySkzSMX53D8bY8xKQ0llBa80MdzYOMTN?usp=sharing">Slides, notebooks and recordings</a> </td>
  </tr>
  <tr>
    <!-- <td class="tg-acii"><span style="color:#333;background-color:#FFF">Week 9 - 28.04</span></td> -->
    <td class="tg-acii"><span style="color:#333;background-color:#FFF;font-weight:bold">Intro to BioImage Analysis and Deep Learning Utilization</span></td>
    <td class="tg-acii"><span style="color:#333;background-color:#FFF">Martin Schatz (Prague)</span></td>
    <td class="tg-acii"></td>
    <td class="tg-acii"><a href="https://drive.google.com/drive/folders/1W-U_5QlJGdDUt6igtDQsveQb-er6Vayp?usp=sharing">Slides, notebooks, recordings</a></td>
  </tr>

  <tr>
    <td class="tg-acii" colspan="5"></td>
  </tr>
  <tr>
    <!-- <td class="tg-acii"><span style="color:#333;background-color:#FFF">14.04 - 31.05</span></td> -->
    <td class="tg-acii" colspan="4"><span style="color:#333;background-color:#FFF;font-weight:bold"><b>Project phase</b></span></td>
  </tr>
  <tr>
    <td class="tg-acii" colspan="5"></td>
  </tr>
  <tr>
    <!-- <td class="tg-kr4b"><span style="font-weight:bold">30.05-1.06</span></td> -->
  <!--   <td class="tg-kr4b"><span style="font-weight:bold">Final meeting Heidelberg</span></td>
    <td class="tg-acii"></td>
    <td class="tg-0pky"><span style="color:#333;background-color:#FFF">The course will end with a 2-day workshop and hackathon meeting in Heidelberg during which students will be able to implement a short project and listen to scientific lectures.</span>
</td> -->
    <td class="tg-0pky"></td>
  </tr>
</tbody>
</table>
